notebooks/ImvizExample.ipynb

```diff
@@ -495,10 +508,10 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# Add 100 randomly generated X,Y (0-indexed) w.r.t. reference image\n",
+    "# Add 20 randomly generated X,Y (0-indexed) w.r.t. reference image\n",
     "# using default marker properties.\n",
-    "t_xy = Table({'x': np.random.randint(0, 4096, 100),\n",
-    "              'y': np.random.randint(0, 2048, 100)})\n",
+    "t_xy = Table({'x': np.random.randint(0, 4000, 20),\n",
+    "              'y': np.random.randint(0, 4000, 20)})\n",
```

Comment:
Less impressive but works better for me. Otherwise, it is still laggy and gives me IOPub errors even with https://github.com/glue-viz/glue-jupyter/pull/325 installed.

```suggestion
    "t_xy = Table({'x': np.random.randint(0, 1000, 10),\n",
    "              'y': np.random.randint(0, 1000, 10)})\n",
```